### PR TITLE
feat(breakout): improve the front-end part for managing breakout duration

### LINKED ISSUE #14384: Manage breakout duration
**Is your feature request related to a problem? Please describe.**
We're looking into combining and simplifying:
- extend duration of ongoing breakout rooms
- reduce duration of ongoing breakout rooms
- set a completely different time for duration of ongoing breakout rooms - i.e. "Let's wrap up, you have 1 minute left" - type scenario

**Describe the solution you'd like**
@tylercopeland has reworked the designs

* drop the existing "Extend breakout time"
* display the duration in the panel as in screenshot
* create a new three dot menu in the breakouts panel. For now let's only list one item - "Manage duration" (the screenshot says Change time but let's go with Manage duration please).
* move End all breakouts to be a menu item "Destroy breakouts" in the three dot menu. This handles the case if the moderator created the breakouts misconfigured and __needs__ to end them right away to reconfigure (record toggle, etc)

Screenshot from 2022-02-16 12-25-43

Screenshot from 2022-02-16 12-29-06

* The Apply should throw a handled error if the time requested is under 1 minute or over the max allowed time (in scenarios where the main room has a limited duration)
Screenshot from 2022-02-16 12-29-18

* if the duration change was successful, show a toast in the main room
Screenshot from 2022-02-16 12-33-07

* if the duration change was successful, show a toast in each breakout and show a message in the public chat of each breakout 
(the Keep Collaborating might be modified later on)
Screenshot from 2022-02-16 12-33-26

Screenshot from 2022-02-16 12-33-20

**Describe alternatives you've considered**
We've tried using "+" and "-" buttons and hitting them many times :)
We also tried to implement a delayed breakout end which only addressed a subset of what was needed #14145 

**Additional context**

### PR DESCRIPTION
**Pull request by JoVictorNunes**
### What does this PR do?

Implements the front-end part as described in https://github.com/bigbluebutton/bigbluebutton/issues/14384. It's a complement of https://github.com/bigbluebutton/bigbluebutton/pull/14421.

**New box for managing breakout duration:**
Captura de tela de 2022-03-01 10-03-17

**New breakout three dot menu:**
Captura de tela de 2022-03-01 10-03-28
Captura de tela de 2022-03-01 10-03-39

### About the toast notifications

This PR doesn't implement the toast notifications for notifying the users about the change of the breakout time. A more sophisticated and centralized mechanism for notifying users is being discussed and implemented (https://github.com/bigbluebutton/bigbluebutton/issues/14493). So, we will be able to reuse that mechanism.

### Closes Issue(s)

Partially Closes #14384

### CODE REVIEW

File: bigbluebutton-html5/imports/api/notifications/server/handlers/breakoutTimeUpdated.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { check } from 'meteor/check';
+import emitter from '../emitter';
+
+export default function ({ body }, meetingId) {
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/api/notifications/server/handlers/breakoutTimeUpdated.js:4]: Please, name the function

File: bigbluebutton-html5/imports/startup/client/base.jsx [hunk 2]
```diff
@@ -108,6 +109,51 @@ class Base extends Component {
     });
     Session.set('isFullscreen', false);
 
+    const notifications = new Mongo.Collection('notifications').find();
```
Comment (Tainan404) [inline bigbluebutton-html5/imports/startup/client/base.jsx:112]: Move finds into a container

File: bigbluebutton-html5/imports/api/notifications/server/handlers/breakoutTimeUpdated.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  const { timeInMinutes } = body;
+
+  check(meetingId, String);
+  check(timeInMinutes, Number);
+
+  const mainRoomMsgKey = 'app.meeting.alertBreakoutTimeExtended.mainRoom';
+  const breakoutRoomMsgKey = 'app.meeting.alertBreakoutTimeExtended.breakout';
+
+  emitter.emit('notification', meetingId, {
```
Comment (ramonlsouza) [inline bigbluebutton-html5/imports/api/notifications/server/handlers/breakoutTimeUpdated.js:13]: please break the .emit part in two (one for main room and another for breakout room), so it becomes more generic and we can reuse it later (one message key = one notification)

**Review by ramonlsouza — CHANGES_REQUESTED** [review]
Duration styles look different than the ones provided in the issue (uppercase, small, gray text + large numbers)

Issue:
Screen Shot 2022-03-01 at 18 07 42

PR:
Screen Shot 2022-03-01 at 18 07 54

File: bigbluebutton-html5/imports/api/notifications/server/handlers/breakoutTimeUpdated.js (hunk 1 above)
Comment (Tainan404) [inline bigbluebutton-html5/imports/api/notifications/server/handlers/breakoutTimeUpdated.js:4]: How can we make sure it doesn't run in both servers, looks for me the back end and front end will emit it

File: bigbluebutton-html5/imports/startup/client/base.jsx (hunk 2 above)
Comment (Tainan404) [reply bigbluebutton-html5/imports/startup/client/base.jsx:112]: Also create a collection with a properly filer structure

File: bigbluebutton-html5/imports/api/notifications/server/handlers/breakoutTimeUpdated.js (hunk 1 above)
Comment (JoVictorNunes) [reply bigbluebutton-html5/imports/api/notifications/server/handlers/breakoutTimeUpdated.js:4]: Actually, I think that only back end will emit it, as the `UpdateBreakoutRoomsTimeEvtMsg` message comes through the `from-akka-apps-redis-channel` channel from akka. It is still a problem, of course, as we need the front end to emit it. I'm working on a temporary solution as a fancier solution is being discussed (https://github.com/bigbluebutton/bigbluebutton/issues/14493).

**Comment by JoVictorNunes** [comment]
> Duration styles look different than the ones provided in the issue (uppercase, small, gray text + large numbers)
> 
> Issue: Screen Shot 2022-03-01 at 18 07 42
> 
> PR: Screen Shot 2022-03-01 at 18 07 54

Fixed!

Captura de tela de 2022-03-04 09-38-54

**Review by ramonlsouza — APPROVED** [review]

**Pull request merged**